src/views/list.ts

```diff
@@ -0,0 +1,83 @@
+/*!
+ * © 2019 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  getListType,
+  JatsStyleType,
+  ManuscriptNode,
+} from '@manuscripts/transform'
+
+import { getActualAttrs } from '../lib/track-changes-utils'
+import { BaseNodeProps } from './base_node_view'
+import BlockView from './block_view'
+import { createNodeOrElementView } from './creators'
+import { EditableBlock } from './editable_block'
+export class ListView<
```

Comment:
I don't like that we're updating `contentDOM` on the fly. For now, let's set `elementType` to ul and the `listStyleType` should take care of the styling. We can check later whether it's possible to update `contentDOM` without copying things around.

Reply:
I have kept the logic to handle two different elements for the list node in case we decide to use it again later, but I have changed the getListType object in order to always retrieve 'ul' element
https://github.com/Atypon-OpenSource/manuscripts-transform/pull/99/commits/3a57caf64a1e22909cfd8a4e67c4d5c4c79a5233